COMMITTERS.adoc

```diff
@@ -0,0 +1,182 @@
+If you forget to do something, miss something, don't worry, it happens.
+
+Do your best and that will already be a lot :).
+
+== Merging Pull Requests
+
+The first thing is that, while being a committer,
+you can't really commit directly to the main repository:
+you need at least one review by your peers.
+
+Thus the need for a pull request.
+
+So you are more like a merger than a committer
+and you might merge more PRs from others than your own work.
+
+While not absolute, here is some advice:
+
+* If you really want a review by a specific person,
+  make it a named reviewer via a GitHub UI
+* If someone asked for a specific reviewer, better
+  give some time for this person to review the PR.
+  If not reviewed in a reasonable amount of time,
+  better ping them again to have some information
+  or ask the original author if we should go ahead.
+* If in doubt (on a specific part or because you're
+  not comfortable reviewing documentation or any other reason),
+  ask for help
+* Except if it's a total no-brainer (typically a typo fix),
+  let the PR bake a few hours so some other people can have a
+  look if interested.
+  If the PR is very large or is a new extension, better let it
+  bake for a few days.
+* We do merge commits so please ask the author to properly
+  squash the commits before merging.
+  The idea is to have proper semantic commits.
+  If the author is not familiar with Git or not available,
+  you can also (carefully)
+  https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork[do it for them]
+  as long as you agreed on it.
+* Make sure the commit comments are meaningful.
+* As a new committer, you can start by giving your approval and not merging
+  and see how things evolve.
+  This should give you some hints of important things to check in the future.
+* Only merge pull requests that target master.
+  **Dealing with old branches is a separate process.**
+* If you are sure a PR should go in and is just waiting for CI,
+  just use the `triage/waiting-for-ci` label.
+  It is a good hint for other committers.
+* If you think a pull request is an essential part of a given milestone
+  (and can realistically be part of it),
+  you should affect this milestone to it so that we can be sure it stays on the radar.
+
+Obviously, each situation is different so use your own judgement,
+and if in doubt, just ask for advice, the other committers are here to help.
+
+== Pull Request Title
+
+The title should be in English, should not contain an issue number.
+It should also not contain ellipsis.
+
+If your commit message was too long and GitHub automatically cut
+the title, that's helpful if you can take the time to move the cut
+part where it belongs to have a full title.
+
+Titles are included in the Release notes so they are important.
+
+A good title would look like: `Fix XXX in Quartz extension` or
```

Comment:
Proposal.
If XXX is an issue number I don't think it is meaningful in a changelog.
`Fix #454 Off by one error in days in Quartz` feels better?

Reply:
I just realized that the description above says that the title should not contain the issue number but here we're saying that a good title would have it?

Reply:
Or I think the XXX means the fix in question: `Fix off by one error in days in Quartz`

Reply:
Better substitute the XXX with something meaningful to avoid any doubts :)

Reply:
Yes, done.